frontend/packages/dev-console/src/components/topology/data-transforms/transform-utils.ts

```diff
@@ -277,9 +277,9 @@ export const addToTopologyDataModel = (
   graphModel: Model,
   dataModelDepicters: TopologyDataModelDepicted[] = [],
 ) => {
-  graphModel.edges.push(...newModel.edges);
+  graphModel.edges.push(...(newModel?.edges || []));
```

Comment:
What you have works, but maybe better to be explicit:
```suggestion
  if (newModel?.edges) {
    graphModel.edges.push(...newModel.edges);
  }
```